frame/claims/src/lib.rs

```diff
@@ -27,10 +27,10 @@ mod types {
 use codec::{Decode, Encode};
 // --- substrate ---
 #[cfg(feature = "std")]
-use frame_support::debug::error;
+use frame_support::{debug::error, traits::WithdrawReasons};
 use frame_support::{
 	ensure,
-	traits::{Currency, ExistenceRequirement::KeepAlive, Get, WithdrawReasons},
+	traits::{Currency, ExistenceRequirement::KeepAlive, Get},
 	{decl_error, decl_event, decl_module, decl_storage},
 };
```

Comment:
It may be better within one `use`.
```rust
 use frame_support::{ 
        debug::error,
 	ensure, 
 	traits::{Currency, ExistenceRequirement::KeepAlive, Get, WithdrawReasons}, 
 	{decl_error, decl_event, decl_module, decl_storage}, 
};
```

Reply:
No, the `use frame_support::{debug::error, traits::WithdrawReasons};` was under the `#[cfg(feature = "std")`.